docker/hail-ubuntu/hail-pip-install

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -ex
+
+python3 -m pip install "$@"
```

Comment:
Is this guaranteed to only take the arguments in the statement we are working with. i.e.

```
hail-pip-install a b c && rm f
```

will only work give hail-pip-install the arguments a, b, and c

Reply:
Yes, in Bash, [&&, ||, and ;](https://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html) are some of the operators which terminate a command. This gets parsed by bash as:
```
COMMAND1 && COMMAND2
```
where `COMMAND1` is  `hail-pip-install a b c`. `$@` refers to the arguments to the currently executing script, so in this case `a`, `b`, and `c`

Reply:
That's what I thought, but I just wanted to make sure.